docker/supervisord.conf

```diff
@@ -39,5 +35,12 @@ command=caddy run --config /etc/caddy/Caddyfile --adapter caddyfile
 autostart=%(ENV_SUPERVISORD_CADDY)s
 autorestart=%(ENV_SUPERVISORD_CADDY)s
 priority=10
-stdout_events_enabled=true
-stderr_events_enabled=true
+stdout_logfile=/dev/fd/1
```

Comment:
What is the point in switching to redirect_stderr instad of the stderr_events_enabled and why did you disable the stdout_events_enabled?

Reply:
There's no event listener configured, so `stdout_events_enabled` and `stderr_events_enabled` are pointless.